Scripts/build-archive.sh

```diff
@@ -27,6 +27,26 @@ if [ ! -d "$PRODUCTS_DIR/iOS/AppCenterDistributeResources.bundle" ] || \
   exit 1
 fi
 
+# Verify prefix.
+verify_prefix() {
+
+  # Get values.
+  frameworkPath=$1
+
+  # Get result from command and check result value.
+  if nm -gU $frameworkPath | awk '{ print $3 }' | grep "_OBJC_CLASS_" | grep -v "_OBJC_CLASS_\$_MSAC"; then
+    echo "These files should have a prefix MSAC."
```

Comment:
Not "files". There is no info what framework is affected. I guess it's better to print something like "Verifying that everything in X.framework has the prefix" before checking/printing the list